matrix-sdk-android/src/main/res/values/strings.xml

```diff
@@ -89,6 +89,16 @@
     <string name="notice_widget_modified">%1$s modified %2$s widget</string>
     <string name="notice_widget_modified_by_you">You modified %1$s widget</string>
 
+    <string name="power_level_admin">Admin</string>
+    <string name="power_level_moderator">Moderator</string>
+    <string name="power_level_default">Default</string>
+    <string name="power_level_custom">Custom (%1$d)</string>
+    <string name="power_level_custom_no_value">Custom</string>
+
+    <string name="notice_power_level_changed_by_you">You changed the power level of %1$s.</string>
+    <string name="notice_power_level_changed">%1$s changed the power level of %2$s.</string>
+    <string name="notice_power_level_diff">%1$s from %2$s to %3$s</string>
```

Comment:
You should a comment to help translator to translate this.
ex:
`<!-- First parameter will be a userId or display name, the two last one will be value of power_level_* -->`